dev/io.openliberty.faces.4.0.thirdparty/bnd.bnd

```diff
@@ -26,10 +26,10 @@ Implementation-Version: 4.0.0
 # org.apache.myfaces.buildtools package is from myfaces-builder-annotations-1.0.9.jar
 Export-Package: \
   org.apache.myfaces.renderkit.html;thread-context=true;version=${Implementation-Version}, \
-  org.apache.myfaces.shared.config;version=${Implementation-Version}, \
-  org.apache.myfaces.shared.renderkit;version=${Implementation-Version}, \
-  org.apache.myfaces.shared.renderkit.html;version=${Implementation-Version}, \
-  org.apache.myfaces.shared.renderkit.html.util;version=${Implementation-Version}
+  org.apache.myfaces.config;version=${Implementation-Version}, \
+  org.apache.myfaces.renderkit;version=${Implementation-Version}, \
+  org.apache.myfaces.renderkit.html.base;version=${Implementation-Version}, \
+  org.apache.myfaces.renderkit.html.util;version=${Implementation-Version}
```

Comment:
@pnicolucci 
For awareness: I'm leaving https://github.com/OpenLiberty/open-liberty/issues/22204 open as i still need to enable our third party tests, and we may need to update the packages upstream since I can't include single class (I'm not aware of a way). `org.apache.myfaces.config` includes more classes than in 2.3 / 3.0.